### PR TITLE
Feature/oks

### PR DESCRIPTION
**Pull request by glehmannmiotto**

### CODE REVIEW

**Comment by glehmannmiotto** [comment]
simple merge

File: CMakeLists.txt
Comment (gcrone) [inline CMakeLists.txt:1]: Missing `find_package(coredal REQUIRED)` and `find_package(appdal REQUIRED)`

**Review by gcrone — CHANGES_REQUESTED** [review]
I think CMakeLists.txt needs fixing. I tried to push an update myself but I don't have permission on this repo.

File: CMakeLists.txt
Comment (glehmannmiotto) [reply CMakeLists.txt:1]: the dependency comes in through readoutlibs/fdreadoutlibs, no?

Comment (gcrone) [reply CMakeLists.txt:1]: Ahh, I hadn’t thought of that. I tried checking it out and saw the configure fail but I hadn’t checked out the other packages.

**Review by gcrone — APPROVED** [review]
All good

**Review by eflumerf — APPROVED** [review]
LGTM

**Pull request merged**